src/main/java/emissary/util/IntegerUtil.java

```diff
@@ -0,0 +1,74 @@
+package emissary.util;
+
+public class IntegerUtil {
+    private IntegerUtil() {}
+
+    // Based on java.lang.Integer.parseInt()
```

Comment:
This method should have a Javadoc comment explaining how it differs from `Integer.parseInt(String)`  (i.e., its support for substrings).